libs/net/dal/Services/AVOverviewTemplateService.cs

```diff
@@ -83,6 +83,8 @@ public override AVOverviewTemplate Update(AVOverviewTemplate entity)
                 originalSubscriber.IsSubscribed = s.IsSubscribed;
             }
         });
+        entity.SubscribersManyToMany.Clear();
```

Comment:
This will result in defragmentation in the database due to deleting and readding records.
The code above deals with adding/updating/removing subscribers.

Reply:
In this case it won't result in deleting and readding records actually, and it will only update the record(s) which we have touched. I can do a quick demo about it if needed, thanks.

Reply:
Updated, thanks Kyle for helping look into this together.